test/metabase/models/database_test.clj

```diff
@@ -216,6 +216,24 @@
                   (is (nil? (db/select-one Secret :id secret-id))
                       (format "Secret ID %d was not removed from the app DB" secret-id)))))))))))
 
+(deftest user-may-not-update-sample-database-test
+  (binding [api/*current-user-id* (mt/user->id :crowberto)]
```

Comment:
is this binding important for the test?

if you do need it, you can use the `with-test-user` macro instead

Reply:
Good catch! I don't think the user is related to the test at all.